interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,84 @@
+import controller.volunteer_controller as vc
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
+    def __init__(self):
+        super(EditVolunteerMenu, self).__init__()
+
+    def edit_volunteer_menu(self) -> None:
+        print("""
+         Select an item you'd like to edit:
+         --------
+         1) First name
+         2) Last name
+         3) Phone number
+         4) Camp
+         5) Availability
+
+         R) Return to previous page
+         0) Log-out
+         """)
+
+    def precmd(self, option: str) -> int:
+        edit_volunteer_options = {"logout": 0,
+                                  "edit_firstname": 1,
+                                  "edit_lastname": 2,
+                                  "edit_phone": 3,
+                                  "edit_camp": 4,
+                                  "edit_availability": 5,
+                                  "return": "r"}
+
+        ### Add input error handling
+        if option.isdigit() and 0 <= int(option) <= 5:
+            return list(edit_volunteer_options.keys())[list(edit_volunteer_options.values()).index(int(option))]
+        elif option == "r" or "R":
+            return list(edit_volunteer_options.keys())[list(edit_volunteer_options.values()).index(option.lower())]
+        else:
+            pass
+
+    def preloop(self) -> None:
+        self.edit_volunteer_menu()
+
+    # ----- basic commands for volunteer account management -----
+    def do_edit_firstname(self):
+        username = input("Username: ")
```

Comment:
I think we need to differentiate between admins and volunteers here?
Cuz volunteers can only edit their own profiles, but admins can choose who to edit.
For volunteers, we need to fetch the user from the session, and dont let them change it.

Reply:
I've added a function to identify user role and then return username in a different way as output.